src/components/StartingPage.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta
+			http-equiv="X-UA-Compatible"
+			content="IE=edge"
+		/>
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
+		/>
+		<link
+			rel="stylesheet"
+			href="../styles/root.css"
+		/>
+		<title>Quiz App</title>
+
+		<style
+			src="./styles/root.css"
+			rel="stylesheet"
+		></style>
+	</head>
+	<body>
+		<div
+			class="padding-small-card shade classA translateOne"
+		>
+			<img
```

Comment:
Your indentation looks off. Please check your prettier extension setting, as I am also using prettier and it doesnt format like this. Check your tab spacing once as well in VS code

Reply:
Okay, I'll do that.